src/jump_input.jl

```diff
@@ -0,0 +1,298 @@
+end
+
+
+function index_to_row_link(
+    model::MOI.FileFormats.MPS.Model)
+
+    i = 0
+    dict = Dict{MOI.ConstraintIndex, Int}()
+    for (S, _) in MOI.FileFormats.MPS.SET_TYPES
+        for ci in MOI.get(
+            model,
+            MOI.ListOfConstraintIndices{MOI.ScalarAffineFunction{Float64}, S}(),
+        )  
+            dict[ci] = i
+            i += 1
+        end
+    end
+    return dict
+end
+
+
+function index_to_column_link(
+    model::MOI.FileFormats.MPS.Model)
+
+    variables = MOI.get(model, MOI.ListOfVariableIndices())
+    return Dict{MOI.VariableIndex,Int}(
+        x => i - 1 for (i, x) in MOI.enumerate(variables)
+    )
+end
+
+
+function write_auxillary_file(
+    new_model::MOI.FileFormats.MPS.Model,
+    lower_variables::Vector{MOI.VariableIndex},
+    lower_objective::MOI.ScalarAffineFunction,
+    lower_constraints::Vector{MOI.ConstraintIndex},
+    lower_sense::MOI.OptimizationSense,
+    aux_address::AbstractString
+    )
+
+    Row = index_to_row_link(new_model)
+    Col = index_to_column_link(new_model)
+
+    open(aux_address, "w") do io 
+        println(io, "N $(length(lower_variables))")
+        println(io, "M $(length(lower_constraints))")
+
+        for x in lower_variables
+            println(io, "LC $(Col[x])")
+        end
+
+        for y in lower_constraints
+            println(io, "LR $(Row[y])")
+        end
+        
+        obj_coefficients = Dict{MOI.VariableIndex,Float64}(
+            x => 0.0 for x in lower_variables
+        )
+        
+        for term in lower_objective.terms
+            if haskey(obj_coefficients, term.variable_index)
+                obj_coefficients[term.variable_index] += term.coefficient
+            end
+        end
+        
+        for x in lower_variables
+            println(io, "LO $(obj_coefficients[x])")
+        end
+        
+        println(io, "OS ", lower_sense == MOI.MAX_SENSE ? -1 : 1)
+    end
+end
+
+
+
+function Writing_MibS_inputs(
+    model::BilevelModel,
+    name::AbstractString = "model"
+)
+
+    new_model, lower_variables, lower_objective, lower_constraints, lower_sense  = _build_single_model(model)
+
+    # Write the MPS
+    #name = "modelv2"
+    name_mps = string(name, ".mps")
+    name_aux = string(name, ".aux")
+
+    MOI.write_to_file(new_model, name_mps)
+    write_auxillary_file(new_model, lower_variables, lower_objective, lower_constraints, lower_sense, name_aux)
+
+end
+
+function Running_MibS(
+add_mpx::AbstractString = "model.mps",
+add_aux::AbstractString = "model.aux"
+)
+    println("============================")
+    println("============================")
+    println("============================")
+    println(add_mpx)
+    println(add_aux)
+    println("============================")
+    println("============================")
+    println("============================")
+    sleep(3)
+    MibS_jll.mibs() do exe
+        run(`$(exe) -Alps_instance $(add_mpx) -MibS_auxiliaryInfoFile $(add_aux)`)
+    end
```

Comment:
Use the form I described in #124 with `pipeline` and the `IOBuffer`. We need to be able to parse the solution.